core-ui/src/main/java/com/anytypeio/anytype/core_ui/features/editor/holders/other/Title.kt

```diff
@@ -64,6 +64,9 @@ sealed class Title(view: View) : BlockViewHolder(view), TextHolder {
             focus(item.isFocused)
         }
         content.pauseTextWatchers {
+            if (item.hint != null) {
```

Comment:
what if you want to clear hint here? maybe change it to `if(content.hint != item.hint)`?

Reply:
To clear the hint just send empty string